realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -47,6 +47,9 @@
     private static final long PRIMARY_KEY_FIELD_COLUMN_INDEX = 1;
     public static final long NO_PRIMARY_KEY = -2;
 
+    public static final int MAX_BINARY_SIZE = 0xFFFFF8 - 8/*array header size*/;
```

Comment:
Where did you get this information from? Perhaps point to that (I assume Core?)

Reply:
@cmelchior See https://github.com/realm/realm-java/pull/5263/files#diff-92306bfcbfe23e84db32c9f0a63c7c62R35